src/index.html

```diff
@@ -11,7 +11,179 @@
+          type="text"
+          id="name"
+          name="name"
+          autocomplete="off"
+          required
+          >
+        </div>
+        <div class="inpt">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </div>
+        <div
+          class="inpt">
+          <label for="birth">Full date of birth:</label>
+          <input
+            type="date"
+            id="birth"
+            name="birth"
+          >
+        </div>
+        <div>
+          <label for="accept">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="accept"
+            name="accept"
+            required
+          >
+        </div>
+      </fieldset>
+      <!-- Registration --------------------- -->
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <div class="inpt">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </div>
+        <div>
+          <label for="pass">Password:</label>
+          <input
+          type="password"
+          id="pass"
+          name="pass"
+          minlength="6"
+          maxlength="16"
+          required
+          >
+        </div>
+      </fieldset>
+      <!-- an interesting facts --------------------- -->
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <div class="inpt">
+          Do you love cats?
+          <input
+            type="radio"
+            id="catsyes"
+            name="cats"
+            value="yes"
+          >
+          <label for="catsyes">Yes</label>
+          <input
+            type="radio"
+            id="catsno"
+            name="cats"
+            value="no">
+          <label for="catsno">No</label>
+        </div>
+        <div class="inpt">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            id="color"
+            name="color"
+          >
+        </div>
+        <div class="inpt">
+          <label for="time">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="time"
+            name="time"
+            step="2"
+          >
+        </div>
+        <div class="inpt">
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+
+          </select>
```

Comment:
```suggestion
            <option value="lada">Lada</option>
          </select>
```
left indents only between same-level elements